libraries/common/subscriptions/app.js

```diff
@@ -29,11 +35,17 @@ export default function app(subscribe) {
 
     // Suppress errors globally
     pipelineManager.addSuppressedErrors([
```

Comment:
As discussed, this should be moved into login action.

Reply:
done